cmake/Platform/System/LinuxDistDetector.cmake

```diff
@@ -6,6 +6,11 @@ function(detect_host_linux_distribution)
         message(AUTHOR_WARNING "Linux distribution detection called on Apple platform")
     else () # Linux host
         find_program(lsb_release_exec lsb_release)
+
+        if ("lsb_release_exec-NOTFOUND" STREQUAL "${lsb_release_exec}")
+            message(FATAL_ERROR "`lsb_release' not found")
```

Comment:
`FATAL_ERROR` fails the whole CMake process, which is something we don't really want to do here since the dist-detection module is not part of the framework's core, just a nice-to-have feature 😃 
Instead, use 'WARNING' to inform the user that it won't benefit from this module until it installs missing dependencies.

Besides, I think it'd be more informative if the message also stated that "Linux distribution couldn't be detected", then the exact reason why.

Reply:
Thanks for the fast feedback, I'll update my pull request!